### DISCUSSION TITLE
How do I exclude a image from pinDigest best practices?

### DISCUSSION THREAD

**Discussion by Tomnivore** [renovatebot/renovate#33150] [Request Help] [answered]
### How are you running Renovate?

None

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

Hello. For base info I'm using Renovate's [best practices presets ](https://docs.renovatebot.com/presets-config/#configbest-practices)  which includes https://docs.renovatebot.com/presets-docker/#dockerpindigests.

I use an image that is built every evening, it has only maven d ependency check and downloads the latest NVD updates to reduce the API calls. The idea is to use its latest tag in the pipelines and you can probably see that it's problematic to tag an image that changes every day.

I'm wondering how I could create a rule that overrides pinDigests for this image. An example would be very appriciated. Let's say the image is named gitlab.myserver.com/group/subgroup/dependency-check-image:latest

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
```json
{
  "matchPackageNames": ["gitlab.myserver.com/group/subgroup/dependency-check-image"],
  "pinDigests": false
}
```

**Reply by Tomnivore** [reply]
Thanks. I had the same but with datasource docker, but I'm guessing I've misunderstood that part. 😄 I'll return to mark this as answered as soon as I confirm this version works.
